docs/DeveloperGuide.md

```diff
@@ -257,71 +257,92 @@ _{Explain here how the data archiving feature will be implemented}_
+Priorities: High (must have) - `H`, Medium (nice to have) - `M`, Low (unlikely to have) - `L`
 
 | Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| ---- | ------------------------------------------ | ------------------------------ | -------------------------------------------------------------------------- |
+| `H`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                     |
+| `H`  | user                                       | add a property                 | keep track of the property                                                 |
+| `H`  | user                                       | delete a property              | stop keeping track of the property                                         |
+| `H`  | user                                       | list all properties            | easily see all properties                                                  |
+| `H`  | user                                       | find property by name          | locate details of property without having to go through the entire list    |
+| `H`  | user                                       | update property tags           | provide details about a property                                           |
+| `H`  | user                                       | find property by tags          | easily see related properties with a given tag                             |
+| `H`  | user                                       | add, update, delete a buyer    | keep track of the buyer                                                    |
```

Comment:
Should we break `add, update, delete a buyer` into 3 separate user stories?

Reply:
Sure thing.